src/pages/index.js

```diff
@@ -122,43 +122,53 @@ class Index extends React.Component {
                 Astronomy Picture of the Day
               </a> is one of NASA's most popular websites. Each day a different image highlighting our universe with a brief description will be featured.
             </p>
-            <header className="major">
-              <code>/spacebot apod</code> Display today's APOD image
+            <p className="major">
+              <code>/spacebot apod</code>
+              Display today's APOD image
+              <br />
```

Comment:
I think all the double <br /> tags is adding too much whitespace. We should go back to the single br tag like was there.